src/Compilers/CSharp/Test/Symbol/Symbols/Retargeting/NoPia.cs

```diff
@@ -1367,6 +1367,227 @@ public void GenericsClosedOverLocalTypes1_1()
         }
 
         [ClrOnlyFact]
+        public void ValueTupleWithMissingImportedType()
```

Comment:
>Imported [](start = 41, length = 8)

I believe "Imported" is somewhat inaccurate term in this context, "Canonical" would reflect the scenario better.